params/config.go

```diff
@@ -491,6 +493,15 @@ func (o *OptimismConfig) String() string {
 	return "optimism"
 }
 
+type CeloConfig struct {
+	EIP1559BaseFeeFloor uint64 `json:"eip1559BaseFeeFloor"`
+}
+
+// String implements the stringer interface, returning the celo config details.
+func (o *CeloConfig) String() string {
+	return "celo"
```

Comment:
Is it some sort of a weird convention to just return this instead of actual details?

Reply:
There is plenty of precedent in the same file: https://github.com/celo-org/op-geth/blob/715ccb804815f2cbee46c773ec4cbe2f32b50b2d/params/config.go#L466-L492

But it is a bit strange.

Reply:
Yes, I just mimicked from the Optimism one, but maybe they also dragged a bad convention. I'll add it (for us at least)